test/suite/test_chunkcache02.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import os
+import random
+import threading
+import time
+import wiredtiger, wttest
+
+from wtdataset import SimpleDataSet
+from wtscenario import make_scenarios
+
+class test_chunkcache(wttest.WiredTigerTestCase):
+    uri = "table:test_chunkcache02"
+
+    format_values = [
+        ('column', dict(key_format='r', value_format='S')),
+        ('row_string', dict(key_format='S', value_format='S')),
+    ]
+
+    cache_types = [
+        ('in-memory', dict(chunk_cache_extra_config='type=DRAM')),
+        ('on-disk', dict(chunk_cache_extra_config='type=FILE,storage_path=/home/ubuntu/chunk-cache-tmp')),
```

Comment:
Etienne made a good comment about the pathing on my workgen test - we may want to make it compatible to other users / evergreen and use something like os.getcwd().

Reply:
Very true, thanks for pointing this out!

Reply:
Updated in [b3a6e7b](https://github.com/wiredtiger/wiredtiger/pull/9507/commits/b3a6e7bdde3583cc7da4bba7a9d44301cabe9075)